cron/duo_logpull.py

```diff
@@ -129,55 +135,73 @@ def process_events(mozmsg, duo_events, etype, state):
 
 def main():
     try:
-        state = pickle.load(open(options.statepath, 'rb'))
+        state = pickle.load(open(options.statepath, "rb"))
     except IOError:
         # Oh, you're new.
         # Note API v2 expect full, correct and within range timestamps in millisec so we start recently
         # API v1 uses normal timestamps in seconds instead
-        state = {'administration': 0, 'authentication': 1547000000000, 'telephony': 0}
+        state = {
```

Comment:
Added default offsets for all categories - only authentication really supports it, but this avoid running into compat issues with the file